Desktop/exellenteam_python/its_the_way.py

```diff
@@ -0,0 +1,9 @@
+
+import os
+path=input("enter a path of directory: ")
+list_of_files=os.listdir(path)
+deep_file=[]
+for f in list_of_files:
+    if len(f)>4 and f[:4]=='deep':
```

Comment:
Perhaps we could make it easier using the .startswith method - search it in google.